app/views/feeds/_tabs.html.erb

```diff
@@ -14,7 +14,15 @@
   </div>
   <% if policy(podcast).update? %>
     <div class="fixed-bottom col-sm-12 col-md-3 col-xl-2">
-      <%= link_to "Add a Feed", new_podcast_feed_path(podcast), class: "btn btn-success d-flex sticky-md-bottom m-4 shadow" %>
+      <div class="btn-group dropup d-flex sticky-md-bottom m-4 shadow">
```

Comment:
This 2nd Add-Apple-Feed dropup button should show _if_ you don't already have an apple feed.

If you already have one, just show the plain "add a feed" button".